.github/workflows/integration.yml

```diff
@@ -15,12 +15,10 @@ jobs:
     timeout-minutes: 15
     runs-on: ubuntu-20.04
     steps:
+      - uses: linkerd/dev/actions/setup-tools@v38
```

Comment:
@stevej and there's an action that unpacks the devcontainer's tools for use in integration tests.